modules/qtwidgets/include/modules/qtwidgets/properties/propertysettingswidgetqt.h

```diff
@@ -363,89 +359,70 @@ class TemplateMinMaxPropertySettingsWidgetQt : public QDialog, public PropertyWi
+    const std::array<QString, 6> labels = {"Min", "Start", "End", "Max", "Separation", "Increment"};
     for (size_t i = 0; i < labels.size(); ++i) {
         gridLayout->addWidget(new QLabel(labels[i], this), 0, static_cast<int>(i));
     }
-    const std::array<char, 4> desc = {'x', 'y', 'z', 'w'};
-    const uvec2 components = OrdinalProperty<T>::getDim();
 
-    int count = 0;
-    for (size_t i = 0; i < components.x; i++) {
-        for (size_t j = 0; j < components.y; j++) {
-            std::stringstream ss;
-            ss << desc[i] << (components.y == 1 ? "" : (std::string{", "} + desc[j]));
-            settings_.push_back(std::make_unique<SinglePropertySetting>(this, ss.str()));
-            gridLayout->addWidget(settings_[count]->label_, count + 1, 0);
-
-            for (int k = 0; k < 6; ++k) {
-                QLineEdit* edit = settings_[count]->addField();
-                gridLayout->addWidget(edit, count + 1, k + 1);
-            }
-            count++;
-        }
+    for (int k = 0; k < 6; ++k) {
+        QLineEdit* edit = setting_.addField();
+        gridLayout->addWidget(edit, 1, k);
     }
 
-    gridLayout->addWidget(btnApply_, count + 1, 0, 1, 1);
-    gridLayout->addWidget(btnOk_, count + 1, 1, 1, 2);
-    gridLayout->addWidget(btnCancel_, count + 1, 3, 1, 2);
+    gridLayout->addWidget(btnApply_, 2, 3);
+    gridLayout->addWidget(btnOk_, 2, 4);
+    gridLayout->addWidget(btnCancel_, 2, 5);
     gridLayout->setColumnStretch(2, 2);
 
     setLayout(gridLayout);
 
-    connect(btnApply_, &QPushButton::clicked, this,
-            &TemplateMinMaxPropertySettingsWidgetQt<T>::apply);
-    connect(btnOk_, &QPushButton::clicked, this, &TemplateMinMaxPropertySettingsWidgetQt<T>::save);
-    connect(btnCancel_, &QPushButton::clicked, this,
-            &TemplateMinMaxPropertySettingsWidgetQt<T>::cancel);
+    connect(btnApply_, &QPushButton::clicked, this, &MinMaxPropertySettingsWidgetQt<T>::apply);
+    connect(btnOk_, &QPushButton::clicked, this, &MinMaxPropertySettingsWidgetQt<T>::save);
+    connect(btnCancel_, &QPushButton::clicked, this, &MinMaxPropertySettingsWidgetQt<T>::cancel);
 
     reload();
-    setWindowTitle(QString::fromStdString(property_->getDisplayName().c_str()));
+    setWindowTitle(utilqt::toQString(property_->getDisplayName()));
 }
 
 template <typename T>
-TemplateMinMaxPropertySettingsWidgetQt<T>::~TemplateMinMaxPropertySettingsWidgetQt() {
+MinMaxPropertySettingsWidgetQt<T>::~MinMaxPropertySettingsWidgetQt() {
     if (property_) property_->deregisterWidget(this);
 }
 
 template <typename T>
-void TemplateMinMaxPropertySettingsWidgetQt<T>::apply() {
+void MinMaxPropertySettingsWidgetQt<T>::apply() {
     NetworkLock lock(property_);
 
     using range_type = typename MinMaxProperty<T>::range_type;
 
     // order of values stored in setting_:
     // "Component", "Min Bound", "Start","End", "Max Bound", "MinSeparation", "Increment"
```

Comment:
Comment needs to be updated?

Reply:
why it's the same

Reply:
Thought it was connected to the labels above